src/components/Fight.jsx

```diff
@@ -0,0 +1,13 @@
+import { useLocation } from 'react-router-dom';
+
+export default function Fight() {
+  const location = useLocation();
+
+  return (
+    <div>
+      <img src={location.state.myHero.image.url} alt="héros" />
+      <p>VS</p>
+      <img src={location.state.vsHero.image.url} alt="vshéros" />
```

Comment:
I don't get where you find these variable, if someone can explain it to me it'd be nice